lib/dao.js

```diff
@@ -2325,6 +2514,20 @@ DataAccessObject.prototype.updateAttribute = function updateAttribute(name, valu
 };
 
 /**
+ * Replace a single attribute.
+ * Equivalent to `replaceAttributes({name: value}, cb)`
+ *
+ * @param {String} name Name of property
+ * @param {Mixed} value Value of property
+ * @param {Function} cb Callback function called with (err, instance)
+ */
+DataAccessObject.prototype.replaceAttribute = function replaceAttribute(name, value, options, cb) {
+  var data = {};
+  data[name] = value;
+  return this.replaceAttributes(data, options, cb);
+};
```

Comment:
Ooops, this will delete all other existing attributes and modify the object to have only single attribute provided in args. That's almost certainly not wanted, please delete `replaceAttribute` completely.